### PR TITLE
WIP: Add migration README

### PR DESCRIPTION
**Pull request by robhenley**
Does anyone have thoughts on using [Golang Migrate](https://github.com/golang-migrate/migrate) for database migrations?

### CODE REVIEW

**Comment by ShaunWMusic** [comment]
at my work, we use Goose - https://github.com/pressly/goose
i have not used that one... so I don't have any input

**Comment by robhenley** [comment]
I'm not sure I care either way @ShaunWMusic.  It's just a tool I've used previously so I'm ok with Goose if you care.  Just wanted to make sure everyone was on board before I actually created some migrations.

**Comment by ShaunWMusic** [comment]
no u are good. it doesnt have to be goose

**Comment by robhenley** [comment]
@ShaunWMusic No rush but any thoughts on this migration?

File: migrations/20210305124354_create_initial_tables.up.sql [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    last_name character varying(50),
+    email character varying(255) NOT NULL,
+    email_verified_at date,
+    created_at date,
+    updated_at date,
+    PRIMARY KEY (id),
+    CONSTRAINT users_email_uniq UNIQUE (email)
+);
+
+CREATE TABLE social_accounts
```
Comment (ShaunWMusic) [inline migrations/20210305124354_create_initial_tables.up.sql:14]: i dont know if we need this

**Review by ShaunWMusic — COMMENTED** [review]
nice! thanks for writing up readme

**Comment by ShaunWMusic** [comment]
only had one question other than that LGTM

**Comment by ShaunWMusic** [comment]
@robhenley did this get closed b/c work is in another PR?

**Comment by robhenley** [comment]
@ShaunWMusic Yes, exactly.

**Pull request closed without merging**